requirements.txt

```diff
@@ -2,12 +2,14 @@ Django==1.10.1
 django-cors-headers==1.3.1
 django-filter==1.0.1
 djangorestframework==3.4.7
+djangorestframework-jsonapi==2.1.1
 DukeDSClient==0.2.14
 funcsigs==1.0.2
+inflection==0.3.1
+lando-messaging==0.1
 mock==2.0.0
 pbr==1.10.0
+pika==0.10.0
 PyYAML==3.12
 requests==2.11.1
 six==1.10.0
-git+git://github.com/Duke-GCB/lando-messaging.git
```

Comment:
Unable to install requirements because of this change.
`pip install -r requirements`:
```
Collecting lando-messaging==0.1 (from -r requirements.txt (line 9))
  Could not find a version that satisfies the requirement lando-messaging==0.1 (from -r requirements.txt (line 9)) (from versions: )
No matching distribution found for lando-messaging==0.1 (from -r requirements.txt (line 9))
```
lando-messaging isn't on PyPI.